README.md

```diff
@@ -102,9 +102,47 @@ Clone the repo to get started and follow these steps:
 The docker-compose setup comes with two pre-built Grafana dashboards. One for listing the discrete test runs as a list, and the other for visualizing the results of a specific test run.
 >Note: The dashboards work with the Native Histogram mapping so it is required to enable it.
 
-### Test result dashboard
-![Prometheus dashboard of k6 test result](./images/prometheus-dashboard-k6-test-result.png)
 
 ### Test list dashboard
 ![Prometheus dashboard of k6 test runs](./images/prometheus-dashboard-k6-test-runs.png)
 
```

Comment:
I don't think we need to reverse this, the second screen is more important.

Reply:
ok